src/docfx/build/page/BuildPage.cs

```diff
@@ -172,8 +172,9 @@ private static (List<Error> errors, Schema schema, PageModel model, JObject meta
 
             return (errors, schema, model, metadata);
 
-            object TransformContent(DataTypeAttribute attribute, object value, string jsonPath)
+            object TransformContent(IEnumerable<DataTypeAttribute> attributes, object value, string jsonPath)
             {
+                var attribute = attributes.OrderBy(attr => attr.Order).First();
                 if (attribute is HrefAttribute)
```

Comment:
how about using `if (attributes.Any(attr => attr is HrefAttribute)` just like in `BuildXrefMap`? This way you don't even need the `Order` property

Reply:
how about split the XrefProperty and other attributes, add validation to oter attributes

---
In reply to: [223548820](https://github.com/dotnet/docfx/pull/3479#discussion_r223548820) [](ancestors = 223548820)